doc/api/deprecations.md

```diff
@@ -2650,6 +2650,16 @@ Type: Documentation-only
 The [`crypto.Certificate()` constructor][] is deprecated. Use
 [static methods of `crypto.Certificate()`][] instead.
 
+### DEP0147: `Permissive rmdir recursive is deprecated, use rm recursive`
```

Comment:
Would "recursive rmdir" and "recursive rm" be more clear to the average reader?

I think a semicolon is called for here?

```suggestion
### DEP0147: `Permissive recursive rmdir is deprecated; use recursive rm`
```